agent/app/agent_capability.go

```diff
@@ -434,6 +435,27 @@ func dependenciesExist(dependencies map[string][]string) (bool, error) {
 	return true, nil
 }
 
+func checkAnyValidDependency(dependencies map[string][]string)(bool, error){
+	var validDependencies = 0
+	for directory, files := range dependencies {
+		if exists, err := pathExists(directory, true); err != nil || !exists {
+			continue
+		}
+
+		for _, filename := range files {
+			path := filepath.Join(directory, filename)
+			if exists, err := pathExists(path, false); err != nil || !exists {
+				continue
```

Comment:
looks like with this continue, the below `validDependencies` will always be incremented even when binaries are not present. you probably want to break and increment only when all binaries are present.

Reply:
good point, will change it.